CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# Optimizely Csharp SDK Changelog
+
+## 3.3.0
+September 3rd, 2019
+
+### New Features:
+- Added support for event batching via the event processor.
+- Events generated by methods like `activate`, `track`, and `isFeatureEnabled` will be held in a queue until the configured batch size is reached, or the configured flush interval has elapsed. Then, they will be combined into a request and sent to the event dispatcher.
+- To configure event batching, set the `eventBatchSize` and `eventFlushInterval` properties in the `OptimizelyFactory` using `OptimizelyFactory.SetBatchSize(int batchSize)` and `OptimizelyFactory.SetFlushInterval(TimeSpan flushInterval)` and then create `OptimizelyFactory.NewDefaultInstance`.
+- Event batching is enabled by default. `eventBatchSize` defaults to `10`. `eventFlushInterval` defaults to `30000` Milliseconds.
+- Updated the `dispose` method representing the process of closing the instance. When `dispose` is called, any events waiting to be sent as part of a batched event request will be immediately batched and sent to the event dispatcher.
+- If any such requests were sent to the event dispatcher, `close` waits for provided `TimeoutInterval` before closing, so that events get successfully dispatched.
+
+### Deprecated
+- `EventBuilder` was deprecated and now we will be using `UserEventFactory` and `EventFactory` to create logEvents.
+- `LogEvent` was deprecated from `TrackNotification` and `ActivateNotification` notifications in favor of explicit `LogEvent` notification.
```

Comment:
looks bit confusing. We are talking about notification and above this point, we are talking about class. Need to distinguish clearly between these two points.